src/atomate2/vasp/flows/lobster.py

```diff
@@ -141,49 +136,38 @@ def make(
         )
         jobs.append(basis_infos)
 
-        # Maker needs to be updated here. If the job itself is updated,
-        # no further updates on the job are possible
-        vaspjob = update_user_incar_settings_maker(
+        # Maker needs to be updated here. If the job itself is updated, no further
+        # updates on the job are possible
+        lobster_static = update_user_incar_settings_maker(
             self.lobster_static_maker,
             basis_infos.output["nbands"],
             structure,
             prev_vasp_dir,
         )
+        jobs.append(lobster_static)
+        lobster_static_dir = lobster_static.output.dir_name
+        lobster_static_uuid = lobster_static.output.uuid
 
-        jobs.append(vaspjob)
-
-        static_run_job_dir = vaspjob.output.dir_name
-        static_run_uuid = vaspjob.output.uuid
-
-        lobsterjobs = get_lobster_jobs(
+        lobster_jobs = get_lobster_jobs(
             lobster_maker=self.lobster_maker,
             basis_dict=basis_infos.output["basis_dict"],
-            wavefunction_dir=vaspjob.output.dir_name,
-            optimization_run_job_dir=optimization_run_job_dir,
-            optimization_run_uuid=optimization_run_uuid,
-            static_run_job_dir=static_run_job_dir,
-            static_run_uuid=static_run_uuid,
-            additional_static_run_job_dir=additional_static_run_job_dir,
-            additional_static_run_uuid=additional_static_run_uuid,
+            wavefunction_dir=lobster_static.output.dir_name,
```

Comment:
This looks all fine. I just realized that wavefunction_dir should be the same as static dir. We could probably simply remove it.

Reply:
Done!